code_map.py

```diff
@@ -850,7 +830,8 @@ class CodeMapListener(sublime_plugin.EventListener):
     def reactivate(self):
         global ACTIVE
 
-        if not ACTIVE and get_code_map_view():
+        if get_code_map_view():
+            CodeMapListener.map_group = win().get_view_index(get_code_map_view())[0]
             ACTIVE = True
             win().run_command('synch_code_map')
 
```

Comment:
Simplified the listeners too, but added some new checks to ensure reactivation if it's disabled for some reasons.